### PR TITLE
Changes to bring back to develop because they were pushed directly to production-sfbay-develop-debug

### PR DESCRIPTION
**Pull request by REASY**
Changes which are in `production-sfbay-develop-debug`, but not in `pr…oduction-sfbay-develop` (and not in `develop` too)

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2205)

### CODE REVIEW

File: src/main/java/beam/analysis/plots/RideHailWaitingAnalysis.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ private Double getCategory(double time, List<Double> categories) {
 
     private static final String graphTitle = "Ride Hail Waiting Histogram";
     private static final String xAxisTitle = "Hour";
-    private static final String yAxisTitle = "Count";
+    private static final String yAxisTitle = "Waiting Time (frequencies)";
```
Comment (colinsheppard) [inline src/main/java/beam/analysis/plots/RideHailWaitingAnalysis.java]: Please change this axis label to "Frequency binned by Waiting Time"

**Comment by colinsheppard** [comment]
Approved though made one small request about the change to the plot label...

**Comment by REASY** [comment]
test!

**Comment by REASY** [comment]
Test!

**Comment by REASY** [comment]
test!

**Comment by JustinPihony** [comment]
For posterity since the build might get overwritten:

> 01:53:52 - should allow scheduler to set the first activity *** FAILED *** (10 milliseconds)
> 01:53:52   java.lang.AssertionError: assertion failed: expected: Terminated TestActor[akka://PersonAgentSpec/user/$$g] but got unexpected message CompletionNotice(0,Vector())
> 01:53:52   at scala.Predef$.assert(Predef.scala:223)
> 01:53:52   at akka.testkit.TestKitBase.expectMsgPF(TestKit.scala:439)
> 01:53:52   at akka.testkit.TestKitBase.expectMsgPF$(TestKit.scala:435)
> 01:53:52   at beam.agentsim.agents.PersonAgentSpec.expectMsgPF(PersonAgentSpec.scala:41)
> 01:53:52   at akka.testkit.TestKitBase.expectTerminated(TestKit.scala:453)
> 01:53:52   at akka.testkit.TestKitBase.expectTerminated$(TestKit.scala:452)
> 01:53:52   at beam.agentsim.agents.PersonAgentSpec.expectTerminated(PersonAgentSpec.scala:41)
> 01:53:52   at beam.agentsim.agents.PersonAgentSpec.$anonfun$new$2(PersonAgentSpec.scala:128)
> 01:53:52   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
> 01:53:52   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)

**Comment by REASY** [comment]
@colinsheppard the change https://github.com/LBNL-UCB-STI/beam/pull/2205/files#diff-e1f29a69747e6477eba9e402dc9cc960R452 breaks `PersonAgentSpec.should allow scheduler to set the first activity` because now it stays in that state. Is it really what we want? If yes, I'll fix the test

**Pull request merged**